pkg/predicate/predicate.go

```diff
@@ -158,7 +158,8 @@ func HasName(name string) predicate.Predicate {
 // HasOperationAnnotation is a predicate for the operation annotation.
 func HasOperationAnnotation() predicate.Predicate {
 	return FromMapper(MapperFunc(func(e event.GenericEvent) bool {
-		return e.Meta.GetAnnotations()[v1beta1constants.GardenerOperation] == v1beta1constants.GardenerOperationReconcile
+		return e.Meta.GetAnnotations()[v1beta1constants.GardenerOperation] == v1beta1constants.GardenerOperationReconcile ||
+			e.Meta.GetAnnotations()[v1beta1constants.GardenerOperation] == v1beta1constants.GardenerOperationMigrate
```

Comment:
As we talked: `GardenerOperationMigrate` should be `GardenerOperationRestore`

Reply:
> Generally for the OperationAnnotationWrapper, which is used by all controllers, the restore annotation should be handled exactly the same as the reconcile annotation

Why using `restore` at all then?

Reply:
Meant to say "all controllers which do not have to restore their state"

Reply:
Migration is replaced by Restore